ui-tests/src/main/java/io/jenkins/plugins/analysis/warnings/IssuesRecorder.java

```diff
@@ -0,0 +1,414 @@
+package io.jenkins.plugins.analysis.warnings;
+
+import java.util.function.Consumer;
+
+import org.jenkinsci.test.acceptance.po.AbstractStep;
+import org.jenkinsci.test.acceptance.po.Control;
+import org.jenkinsci.test.acceptance.po.Describable;
+import org.jenkinsci.test.acceptance.po.Job;
+import org.jenkinsci.test.acceptance.po.PageArea;
+import org.jenkinsci.test.acceptance.po.PageAreaImpl;
+import org.jenkinsci.test.acceptance.po.PostBuildStep;
+
+/**
+ * Page object for the IssuesRecorder of the warnings plugin (white mountains release).
+ *
+ * @author Ullrich Hafner
+ */
+@Describable("Record compiler warnings and static analysis results")
+public class IssuesRecorder extends AbstractStep implements PostBuildStep {
+    private Control toolsRepeatable = findRepeatableAddButtonFor("tools");
+    private Control filtersRepeatable = findRepeatableAddButtonFor("filters");
+    private Control qualityGatesRepeatable = findRepeatableAddButtonFor("qualityGates");
+    private Control advancedButton = control("advanced-button");
+    private Control enabledForFailureCheckBox = control("enabledForFailure");
+    private Control ignoreQualityGate = control("ignoreQualityGate");
+    private Control overallResultMustBeSuccessCheckBox = control("overallResultMustBeSuccess");
+    private Control referenceJobField = control("referenceJob");
+    private Control aggregatingResultsCheckBox = control("aggregatingResults");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Private field 'aggregatingResultsCheckBox' could be made final; it is only initialized in the declaration or constructor.](https://app.codacy.com/manual/uhafner/warnings-ng-plugin/pullRequest?prid=5070362)